Source/WebCore/html/HTMLFrameSetElement.idl

```diff
@@ -30,6 +30,7 @@
     [WindowEventHandler] attribute EventHandler onload;
     [WindowEventHandler] attribute EventHandler onresize;
     [WindowEventHandler] attribute EventHandler onscroll;
+    [WindowEventHandler, EnabledBySetting=ScrollendEventEnabled] attribute EventHandler onscrollend;
```

Comment:
Same comment here as for HTMLBodyElement

Reply:
Fixed.